src/CodeStyle/CSharp/Tests/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
There's really know way to have this in a single place and add a project file entry that points at it instead? @davkean and @srivatsn seemed to think that it could be a linked file in the project file to a shared location...